edx_rest_framework_extensions/auth/jwt/tests/test_decoder.py

```diff
@@ -143,6 +145,24 @@ def test_upgrade(self):
         upgraded_payload['iat'], upgraded_payload['exp'] = jwt_payload['iat'], jwt_payload['exp']
         self.assertDictEqual(jwt_decode_handler(token), upgraded_payload)
 
+    def test_failure_invalid_signature(self):
+        """
+        Verifies the function logs decode failures with invalid signature,
+        and raises an InvalidTokenError if the token cannot be decoded
+        """
+        # Create tokens using each invalid signature and attempt to decode them against
+        # the valid signature.
+        with mock.patch('edx_rest_framework_extensions.auth.jwt.decoder.logger') as patched_log:
+            with self.assertRaises(jwt.InvalidTokenError):
+                signing_key = 'invalid-secret-key'
+                # Generate a token using the invalid issuer data
```

Comment:
This comment is no longer accurate. You could possibly delete this altogether, but you could also update it.